check_diff/tests/bash_commands.rs

```diff
@@ -0,0 +1,34 @@
+use check_diff::change_directory_to_path;
+use std::env;
+use std::ffi::OsStr;
+use tempfile::Builder;
+use tracing::error;
+
+#[test]
+fn cd_test() {
+    // create directory in current directory
+    let dir = Builder::new().tempdir_in("");
```

Comment:
I don't think we need to use the builder here. Probably simpler to use the [tempdir](https://docs.rs/tempfile/latest/tempfile/fn.tempdir.html) function and just `unwrap` since we're in a test case. If we fail to create the temporary directory then there's not much more we can do.